pyproject.toml

```
@@ -12,20 +12,16 @@ license = "MIT"
 python = "^3.9"
 moviepy = "^1.0.3"
 flask = "<2.3" # Since we rely on the before_first_request decorator (https://flask.palletsprojects.com/en/2.2.x/api/?highlight=before_first_request#flask.Flask.before_first_request)
-# Needed to install gym 0.21. 
-# https://github.com/openai/gym/issues/3176#issuecomment-1408598628
-setuptools = {version = "65.5.0"}
-pip = "21"
-gym = {extras = ["box2d"], version = "^0.21.0", optional = true}
-imitation = { git = "https://github.com/rk1a/imitation", branch = "ecda-experiment", optional = true}
+gymnasium = {extras = ["box2d"], version = "^0.28.1", optional = true}
+imitation = { git = "https://github.com/rk1a/imitation", branch = "human-prefs", rev="4dbf20c6d42286377901dd538cd6c5001225fa40",  optional = true}
```

We should use the latest rev of the branch instead of pinning it. (I know I sent you this change, my mistake.) Please change this and re-run `poetry update`.

```suggestion
imitation = { git = "https://github.com/rk1a/imitation", branch = "human-prefs",  optional = true}
```

---

this rev could be deleted, but came with some minor drawbacks, that have been adjusted in the most recent commit within this pull request